html/changelogs/Sood.yml

```diff
@@ -1,2 +1,4 @@
 author: Sood
-changes: []
+changes:
+    - bugfix: Portable and wall-mounted flashers no longer flash through sunglasses.
+    - bugfix: Players will no longer be selected twice for the same antag role.
```

Comment:
The entire point of an atomic PR is to prevent me shoving features in with bugfixes and such
The changelog entry here that isn't part of this PR is already in the codebase
What do you want, a separate PR for a one-line changelog? That sounds excessive